CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
```

Comment:
So in the planning we have this listed as a 4.3.0 instead of 4.2.1. If you want to update that and put today's date on it, we should be able to get this published today.

Reply:
I've just updated the changelog and package version, should be good to get this out the door today.